thermo/gpumd/io.py

```diff
@@ -249,6 +249,61 @@ def import_trajectory(filename='movie.xyz', in_file=None, atom_types=None):
 # Write Related
 #########################################
 
+def create_kpoints(special_points, npoints, atom_c):
```

Comment:
```suggestion
def create_kpoints(atoms, special_points = 'G', npoints=1):
```

Reply:
Order of the input arguments should match that in the comments below. Usually in ASE, the first parameter is the Atoms object. I have also added some default parameters. Check to make sure that these work.